### ISSUE TITLE
Add our POV on Interface Builder "User Defined Runtime Attributes"

### ISSUE THREAD

**Issue by tylermilner** [BottleRocketStudios/iOS-Project-Standards#21]
I'm talking about this:
![image](https://user-images.githubusercontent.com/1720304/63977657-f4e4ce00-ca79-11e9-81af-23de76ccdfdf.png)

IMO, we should prefer to **not** use these since  it doesn't seem to be a standard approach across the iOS development community. You're essentially writing code inside of Interface Builder, where you don't get any help from the compiler for type checking and typos. It can also be a source of frustration since it's probably the last place you're going to look when trying to hunt down a crash or bug that's creating some weird behavior.

**Comment by rmirabelli** [comment]
I agree 100%

**Comment by wmcginty** [comment]
I agree as well, with the possible exception that anyone using IBInspectable properties is going to be writing to these run time attributes through that process.

**Comment by tylermilner** [comment]
@wmcginty agreed. Forgot about `IBInspectable`, but that's a fair use case.
